### PR TITLE
fix(wiki): very quick fix to the wiki page

### PR DESCRIPTION
**Pull request by gridcat**
I have made a very quick fix to make wiki page to looks like page.
Before:
image

After:
image
Still a lot of tuning must be done. 

Also I've updated the wiki from the original website (English version only so far).

### CODE REVIEW

**Comment by barton2526** [comment]
Ideally this should be two commits for ease of review.
https://deepsource.io/blog/git-best-practices/

File: _layouts/wiki.html [hunk 1]
```diff
@@ -3,27 +3,31 @@
...
+                          </ul>
+                        </div>
+                  </div>
+                  <div class="col-xl-9">
+                      {{ content  }}
+                  </div>
+              </div>
+          </div>
+        </section>
+        </main>
```
Comment (gridcat) [inline _layouts/wiki.html:30]: 🥩

File: assets/css/gridcoin.css [hunk 2]
```diff
@@ -998,3 +998,25 @@ footer {
...
+.docs-menu {
+  margin-top: 0.5rem;
+}
+
+.docs-container pre {
+  border: 1px solid #bbcddb;
+  border-radius: 0.5rem; 
+  background-color: #e9edf1;
+  padding: 1rem;
+}
```
Comment (gridcat) [inline assets/css/gridcoin.css:1022]: 🥩

File: wiki/index.md [hunk 3]
```diff
@@ -148,10 +150,12 @@ scientific computations instead of securing the blockchain.
       - [AUR daemon](https://aur.archlinux.org/packages/gridcoinresearchd/)
 
 <!-- end list -->
-
+  {% comment %} This lists out all the exchanges from _data/exchanges.yml {% endcomment %}
```
Comment (barton2526) [inline wiki/index.md:153]: Interacts with #11

Comment (gridcat) [reply wiki/index.md:153]: ok, let merge the rest first

**Comment by gridcat** [comment]
> Ideally this should be two commits for ease of review. https://deepsource.io/blog/git-best-practices/

agreed

**Comment by barton2526** [comment]
This should not have been merged...

**Comment by barton2526** [comment]
Needs to be reopened @gridcat

**Pull request merged**